libs/ui/elements/src/lib/related-record-card/related-record-card.component.html

```diff
@@ -1,9 +1,11 @@
-<div class="max-w-xs overflow-hidden rounded-lg shadow-lg">
-  <img
-    class="object-cover w-full h-52"
-    [src]="record.thumbnailUrl"
-    [alt]="record.title"
-  />
+<div class="max-w-xs overflow-hidden rounded-lg shadow-lg bg-white">
+  <div class="object-cover w-full h-52 bg-gray-100">
+    <img
+      *ngIf="record.thumbnailUrl"
+      [src]="record.thumbnailUrl | safe: 'url'"
+      [alt]="record.title"
+    />
+  </div>
```

Comment:
I feel like this was already addressed in `gn-ui-record-thumbnail` (having a placeholder in case no thumbnail was given)

Reply:
Yes, so maybe `gn-ui-record-thumbnail` should be refactored to only display the image, there are way to many divs and css classes to be used as it is in other components.

Reply:
it does display the image only: https://github.com/geonetwork/geonetwork-ui/blob/master/libs/ui/search/src/lib/record-thumbnail/record-thumbnail.component.html

Honestly I think you can probably just use it straight away without modification